server/manager/src/main/java/org/apache/accumulo/manager/ManagerClientServiceHandler.java

```diff
@@ -648,9 +649,12 @@ public void requestTabletHosting(TInfo tinfo, TCredentials credentials, String t
       inProgress.forEach(hostingRequestInProgress::remove);
     }
 
-    // ELASTICITY_TODO pass ranges of individual tablets
-    manager.getEventCoordinator().event(success, "Tablet hosting requested for %d tablets in %s",
-        success.size(), tableId);
+    final List<Range> ranges = new ArrayList<>(success.size());
+    success.forEach(ke -> ranges.add(ke.toDataRange()));
+    final List<Range> tabletRanges = Range.mergeOverlapping(ranges);
+    manager.getEventCoordinator().event(success,
+        "Tablet hosting requested for %d tablets in ranges %s in table %s", success.size(),
+        tabletRanges, tableId);
```

Comment:
I think we could just remove the comment and leave this code as is. Looking at the comment, I made it and it was mistake.  I think I saw the table id in the method call and assumed that was causing an entire table scan, however that is not the case the tableId is just for logging.  The first argument to the method `success` does seem to already be individual tablets that need attention.


```suggestion
    manager.getEventCoordinator().event(success, "Tablet hosting requested for %d tablets in %s",
        success.size(), tableId);
```

Reply:
Also, there is existing logging earlier in the method for each tablet that needs to be hosted.